src/taipy/gui/gui.py

```diff
@@ -914,7 +914,8 @@ def _call_user_callback(self, state_id: t.Optional[str], user_callback: t.Callab
                 self.__set_client_id_in_context(state_id)
                 return self._call_function_with_state(user_callback, args)
         except Exception as e:
-            warnings.warn(f"Exception raised in invoke_callback: '{user_callback.__name__}':\n{e}")
+            if not self._call_on_exception(user_callback.__name__, e):
+                warnings.warn(f"invoke_callback: Exception raised in function : '{user_callback.__name__}':\n{e}")
```

Comment:
Error message should be
f"invoke_callback: Exception raised in function '{user_callback.\_\_name\_\_}'.\n{e}")

To match the one in on_init